src/method/method_query_handler.hpp

```diff
@@ -111,13 +116,14 @@ namespace cubmethod
       int prepare_query (prepare_info &info, int &flag);
       int prepare_call (prepare_info &info, int &flag);
 
+      /* check */
+
+
```

Comment:
There is a comment related to check, but no actual execution code.
Any code to add for check?

Reply:
I'm gonna remove it.